libs/sdk-backend-base/src/normalizingBackend/normalizer.ts

```diff
@@ -186,6 +188,22 @@ export class Denormalizer {
     private originalMeasureTitle = (originalDef: IMeasure, nameFromBackend: string): string => {
         return measureAlias(originalDef) || measureTitle(originalDef) || nameFromBackend;
     };
+
+    private originalMeasureFormat = (measure: IMeasure, formatFromBackend: string): string => {
+        const format = isDerivedMeasure(measure)
+            ? this.getDerivedMeasureFormat(measure)
+            : measureFormat(measure);
+        return format || formatFromBackend;
+    };
+
+    private getDerivedMeasureFormat = (derivedMeasure: IMeasure): string | undefined => {
+        const masterMeasureLocalIdentifier = measureMasterIdentifier(derivedMeasure);
+        if (!masterMeasureLocalIdentifier) {
+            return;
+        }
+        const measure = this.originalMeasures[masterMeasureLocalIdentifier];
+        return measureFormat(measure);
+    };
```

Comment:
i think this isn't 100% correct. we want to:

1. get format of the measure
2. if format available, use it. time comparison measures may have explicit format set on them -> denormalizer will use them
2. get master measure identifier
3. if available, get measure, get descriptor for that measure, call originalMeasureFormat measure + format specified for that measure in the descriptor. this can cater for recursive derivation
4. else get value from backend

note 1: normalizer does check for cycles in the measure derivation graph, no need to check it here
note 2: please drop the isDerivedMeasure from model, we don't need it